<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LineWidget</class>
 <widget class="QWidget" name="LineWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>532</width>
    <height>70</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Line 1:</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLineEdit" name="lineEdit"/>
   </item>
   <item>
    <widget class="QPushButton" name="AddDataButton">
     <property name="text">
      <string>Add data</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="removeLineButton">
     <property name="text">
      <string>Remove line</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
